test/Tizen.NUI.Tests/Tizen.NUI.Devel.Tests/testcase/public/BaseComponents/TSTextConstants.cs

```diff
@@ -0,0 +1,280 @@
+﻿using global::System;
+using NUnit.Framework;
+using NUnit.Framework.TUnit;
+using Tizen.NUI.Components;
+using Tizen.NUI.BaseComponents;
+using System.Collections.Generic;
+
+namespace Tizen.NUI.Devel.Tests
+{
+    using tlog = Tizen.Log;
+
+    [TestFixture]
+    [Description("public/BaseComponents/TextConstants")]
+    public class PublicTextConstantsTest
+    {
+        private const string tag = "NUITEST";
+
+        [SetUp]
+        public void Init()
+        {
+            tlog.Info(tag, "Init() is called!");
+        }
+
+        [TearDown]
+        public void Destroy()
+        {
+            tlog.Info(tag, "Destroy() is called!");
+        }
+
+        [Test]
+        [Category("P1")]
+        [Description("InputFilter Equals")]
+        [Property("SPEC", "Tizen.NUI.Text.InputFilter.Equals M")]
+        [Property("SPEC_URL", "-")]
+        [Property("CRITERIA", "MR")]
+        [Property("AUTHOR", "bowon.ryu@samsung.com")]
+        public void InputFilterEquals()
+        {
+            tlog.Debug(tag, $"InputFilterEquals START");
+
+            var inputFilter = new Tizen.NUI.Text.InputFilter();
+            Assert.IsNotNull(inputFilter, "Can't create success object inputFilter");
+            Assert.IsInstanceOf<Tizen.NUI.Text.InputFilter>(inputFilter, "Should be an instance of inputFilter type.");
+
+            var compare = new Tizen.NUI.Text.InputFilter();
+            Assert.AreEqual(true, inputFilter == compare, "Should be true!");
+            Assert.AreEqual(inputFilter.GetHashCode(), compare.GetHashCode(), "Should be true!");
+
+            compare.Accepted = @"[\d]";
+            Assert.AreEqual(true, inputFilter != compare, "Should be true!");
```

Comment:
what about using IsFalse or NotSame?

Reply:
This is to validate the != operator .
I think I can use IsTrue. What do you think?

`Assert.IsTrue(inputFilter != compare, "Should be true!");`

Reply:
that is good to me :)